README.md

```diff
@@ -0,0 +1,58 @@
+# EQ Performance Benchmark
+
+## Summary
+
+This is a performance benchmarking tool designed to measure the performance of [EQ Survey Runner](https://github.com/ONSDigital/eq-survey-runner).
+
+This repository was heavily inspired by the [census performance tests](https://github.com/ONSdigital/census-eq-performance-tests)
+
+The benchmark uses locust to perform a load test. The locust test consists of a set of endpoints which are generated dynamically from a HAR file. This allows us run a functional test, or a manual test representing a normal user's behaviour. The HTTP requests from this test can be exported to a HAR file, which can then be imported into this benchmarking tool. The same requests will be modified slightly and rerun during the locust benchmark.
+
+## Instructions
+
+Open the network inspector in Chrome or Firefox and ensure 'preserve log' is ticked. Run your manual or functional test.
+
+After the test is complete, right-click on one of the requests in the network inspector and save the log as a HAR file. Save this file in the root directory of this project as `requests.har` or alternatively use the path to this HAR file in the `HAR_FILEPATH` environment variable when running locust.
+
+To run a benchmark, use:
+
+```bash
+pipenv install
+pipenv run ./run.sh
+```
+
+This will run 1 minute of locust requests with 1 user and no wait time between requests. It will also process the CSV output to combine `collection_exercise_id` into `[ID]` in the requests.
```

Comment:
Should be 
```suggestion
This will run 1 minute of locust requests with 1 user and no wait time between requests. It will also process the CSV output to combine `collection_exercise_sid` into `[ID]` in the requests.
```